server/routes/issues.js

```diff
@@ -148,4 +148,37 @@ router.post('/:issueid/comment', async (req, res) => {
   return res.send(savedIssue);
 });
 
+router.patch('/:issueid/upvote', async (req, res) => {
+  const { issueid } = req.params;
```

Comment:
here you haven't verified whether it is a mongo id or not.

Reply:
Added now.